README.md

```diff
@@ -10,22 +10,22 @@ It provides an extremely modular and flexible platform for implementing
 transaction-based updates to shared state between
 untrusted parties coordinated by consensus algorithms.
 
-Incubation Notice
------------------
-
-This project is an _active_ Hyperledger project. It was proposed to the 
-community and documented [here](http://bit.ly/1T6eVBH). Information on what 
-_Active_ entails can be found in the 
-[Hyperledger Project Lifecycle document](https://goo.gl/4edNRc).
-
 Documentation
 -------------
 
 The latest documentation for Hyperledger Sawtooth is available within this repo in
 the [docs](docs) subdirectory.
 
 Documentation for our stable release is available at: 
-http://intelledger.github.io/.
+https://sawtooth.hyperledger.org/docs/
+
+Project Status
+-----------------
+
+This project is an _active_ Hyperledger project. It was proposed to the 
+community and documented [here](http://bit.ly/1T6eVBH). Information on what 
+_Active_ entails can be found in the 
+[Hyperledger Project Lifecycle document](https://goo.gl/4edNRc).
```

Comment:
This might be a nitpick: This goo.gl URL goes to an obsolete Project Lifecycle page.  (It simply says "Moved to Project Lifecycle" with a link to https://wiki.hyperledger.org/community/project-lifecycle .)  Should this goo.gl URL be updated to skip the intermediate page?

Reply:
No reason to use the short link here.